### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by nicolbidone**
Screenshot_1554497737

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
+    />
+
+    <android.support.design.widget.FloatingActionButton
+            android:id="@+id/fab_download"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="start|bottom"
+            android:layout_margin="@dimen/fab_size"
+            android:src="@drawable/ic_download_info"
     />
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: put this in the above line

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
     <android.support.design.widget.FloatingActionButton
-            android:id="@+id/fab"
+            android:id="@+id/fab_getStored"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="end|bottom"
             android:layout_margin="@dimen/fab_size"
-            android:src="@drawable/ic_save_info"
+            android:src="@drawable/ic_update_info"
+    />
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: put this in the above line

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (nicolbidone) [reply app/src/main/res/layout/activity_main.xml]: Done

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (nicolbidone) [reply app/src/main/res/layout/activity_main.xml]: Done

**Review by lucianoeli — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**